packages/kitchen-sink/src/examples/paragraph-summerizer.jsx

```diff
@@ -0,0 +1,256 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const paragraphSummerizer = () => {
+  const [value, setValue] = useState('');
+  const [data, setData] = useState([]);
+  const [submitting, setSubmitting] = useState(false);
+  const [isCopy, setIsCopy] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setSubmitting(true);
+
+    const requestOptions = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: 'Bearer ' + import.meta.env.VITE_OPENAI_API,
```

Comment:
I don't think this is working. Cos, it seems like you need an API key that you are not providing in production which could cause the deployed version of this to not work. I'll suggest using another alternative or any provider.


https://github.com/aidenybai/million/assets/96334363/52eb82c1-bb81-4ce9-a5c8-00b4b1b40da2